templates/settings.yml.erb

```diff
@@ -68,7 +68,9 @@
 # Uncomment and modify if you want to change the default pid file '/var/run/foreman-proxy/foreman-proxy.pid'
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
-# HTTP ports configuration
+# host and ports configuration
+# host to bind ports to (possible values: *, localhost, 0.0.0.0)
```

Comment:
I'd leave ports out: ```host to bind on```. Pretty sure you can also include ```::``` for IPv6 and maybe include an example IP?